tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore5IastTests.cs

```diff
@@ -735,6 +735,28 @@ await VerifyHelper.VerifySpans(spansFiltered, settings)
                             .UseFileName(filename)
                             .DisableRequireUniquePrefix();
     }
+
+    [SkippableFact]
+    [Trait("RunOnWindows", "True")]
+    public async Task TestIastReflectionInjectionRequest()
+    {
+        var filename = IastEnabled ? "Iast.ReflectionInjection.AspNetCore5.IastEnabled" : "Iast.ReflectionInjection.AspNetCore5.IastDisabled";
+        if (RedactionEnabled is true) { filename += ".RedactionEnabled"; }
```

Comment:
Since we don't use redaction in this vulnerability, I would move this test to the AspNetCore5IastTestsFullSamplingIastEnabled class (Iast enabled and no evidence redaction) or, at least, use the same snapshot file regardless of the evidence redaction (the content should be the same).

Reply:
Okay! I moved it to the `AspNetCore5IastTestsFullSamplingIastEnabled` class.